docs/2.0/docs/patcher/concepts/patches.md

```diff
@@ -1,3 +1,48 @@
 # What is a Patcher Patch?
 
-This page is under construction
+A Patcher Patch is a set of instructions that Patcher can carry out to resolve a breaking change. 
+This instruction sheet is delivered by means of a `yaml` file in a specific format:
+
+```yaml
+name: "<name-of-patch>"
+description: "<description for patch>"
+author: <your-name-here> <<your-email-address-here>>
+
+# Optional dependencies. Terrapatch is a typical one
+dependencies:
+  - name: terrapatch
+    version: "0.1.0"
+
+# Steps necessary to resolve breaking change
+steps:
+  - name: "<name-of-step>"
+    run: <command-to-run>
+  - name: "<name-of-second-step>"
+    run: <second-command-to-run>
+  # etc
+```
+
+## As a Consumer of Patches...
+
+All that's required to ingest patches for breaking IaC changes is to run a `patcher update` (with whatever flags you deem appropriate). 
+
+This command can be run locally, in which case, the steps outlined in the patch(es) would be executed within your codebase, and it would be up to you to apply IoC changes, or open them in a PR against your codebase, whichever is appropriate for your use case. 
```

Comment:
This sounds confusing as written. Can you more clearly separate the specific things that are happening?

For example:
- `patcher update` executes the steps in the patch
- This changes the code in your local environment, but does not commit it or do any kind of `terragrunt apply` or other operation against it.
- It's up to you to apply the code and open a PR that submits it for review.